bindgen/it/src/it/opa/src/test/java/OpaTest.java

```diff
@@ -0,0 +1,39 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class OpaTest {
+
+    @Test
+    public void opaModule() {
+        // Arrange
+        var opa = new OpaTestModule();
+
+        // Act
+        var ctxAddr = opa.opaEvalCtxNew();
+        var input = "{\"user\": \"alice\"}";
+        var inputStrAddr = opa.opaMalloc(input.length());
```

Comment:
@electrum I haven't checked yet, but I would verify #507 on this kind of examples.